ethereum_history_api/oracles/src/noir/oracles/transactionOracle.int.test.ts

```diff
@@ -0,0 +1,66 @@
+import { describe, expect, it } from 'vitest';
+import { createMockClient } from '../../ethereum/mockClient.js';
+import { OFFSETS, getTransactionOracle } from './transactionOracle.js';
+
+describe('getReceiptOracle', () => {
+  it('success', async () => {
+    const cancunBlockNumberInNoirFormat = '0x12884e1';
+    const chainLinkTransferTxIdInNoirFormat = '0x08';
+    const mockFilePaths = [
+      './fixtures/mainnet/cancun/small_block/eth_getBlockByHash_19432673_includeTransactions.json'
+    ];
+    const client = await createMockClient(mockFilePaths);
+
+    const txWithProof = await getTransactionOracle(client, [
+      [cancunBlockNumberInNoirFormat],
+      [chainLinkTransferTxIdInNoirFormat]
+    ]);
+
+    expect(txWithProof[OFFSETS.TX_TYPE]).toStrictEqual('0x02');
+    expect(txWithProof[OFFSETS.NONCE]).toStrictEqual('0x485a');
+    expect(txWithProof[OFFSETS.GAS_LIMIT]).toStrictEqual('0x0493e0');
+    // prettier-ignore
+    expect(txWithProof[OFFSETS.TO]).toStrictEqual([
+        '0x51','0x49','0x10','0x77','0x1a','0xf9','0xca','0x65','0x6a','0xf8',
+        '0x40','0xdf','0xf8','0x3e','0x82','0x64','0xec','0xf9','0x86','0xca'
+    ]);
+    expect(txWithProof[OFFSETS.TO_IS_SOME]).toStrictEqual('0x01');
+    expect(txWithProof[OFFSETS.VALUE]).toStrictEqual('0x');
+    // prettier-ignore
+    expect(txWithProof[OFFSETS.DATA]).toStrictEqual([
```

Comment:
This has length of 136

Reply:
Corrected, now it has length 68 as it is length in bytes (136 is in nibbles)